shared/override-d.ts/react-redux/index.d.ts

```diff
@@ -227,32 +227,13 @@ export type ResolveArrayThunks<TDispatchProps extends ReadonlyArray<any>> = TDis
 // KB added
 export type ConnectedComponentType<TMergedProps, TOwnProps> = <C extends ComponentType<any>>(
   component: C
-) => TMergedProps extends React.ComponentProps<C> ? ConnectedComponentClass<C, TOwnProps> : never
-
-// To debug why the connect is returning never
-export type ConnectedComponentTypeDEBUG<TMergedProps, TOwnProps> = <C extends ComponentType<any>>(
-  component: C
 ) => TMergedProps extends React.ComponentProps<C>
   ? ConnectedComponentClass<C, TOwnProps>
-  : [
-      'missing props:',
-      Exclude<keyof GetProps<C>, keyof TMergedProps>,
-      'extra props:',
-      Exclude<keyof TMergedProps, keyof GetProps<C>>,
-      GetProps<C>,
-      TMergedProps
-    ]
-
-export interface ConnectDEBUG {
-  <TOwnProps, TStateProps, TDispatchProps, TMergedProps>(
-    mapStateToProps: MapStateToProps<TStateProps, TOwnProps>,
-    mapDispatchToProps: MapDispatchToProps<TDispatchProps, TOwnProps>,
-    mergeProps: MergeProps<TStateProps, TDispatchProps, TOwnProps, TMergedProps>,
-    options?: Options<TypedState, TStateProps, TOwnProps, TMergedProps>
-  ): ConnectedComponentTypeDEBUG<TMergedProps, TOwnProps>
-}
-
-export const connectDEBUG: ConnectDEBUG
+  : {
```

Comment:
this won't show missing props, right? Maybe we could make it `{ missing: Exclude<etc>, present: WhatYouHaveNow }`?

Reply:
good catch, i just include this info in the latest commit changes:

<img width="375" alt="Screen Shot 2020-01-08 at 9 46 33 AM" src="https://user-images.githubusercontent.com/516435/71987275-e5317f00-31fb-11ea-91d2-06bd7e130750.png">